src/ui/ui.qrc

```diff
@@ -10,6 +10,11 @@
         <file>authenticationInApp/ViewAuthenticationUnblockCodeNeeded.qml</file>
         <file>authenticationInApp/ViewAuthenticationVerificationSessionByEmailNeeded.qml</file>
         <file>authenticationInApp/ViewAuthenticationVerificationSessionByTotpNeeded.qml</file>
+        <file>components/VPNControllerView.qml</file>
```

Comment:
We have a guide style to call QML components `VPN`+name. It can change, but do we want to start with this PR?